Source/CarthageKit/Xcode.swift

```diff
@@ -950,6 +950,25 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 	}
 
 	return BuildSettings.SDKsForScheme(scheme, inProject: project)
+		.flatMap(.Concat) { sdk -> SignalProducer<(SDK, BuildSettings), CarthageError> in
+			var argsForLoading = buildArgs
+			argsForLoading.sdk = sdk
+
+			return SignalProducer(value: sdk)
+				.zipWith(BuildSettings.loadWithArguments(argsForLoading))
```

Comment:
Rather than creating a `SignalProducer` for `sdk` and using `zipWith`, you could just use `map`:

```swift
return BuildSettings
    .loadWithArguments(argsForLoading)
    .map { (sdk, $0) }
```